src/GlobalState.lua

```diff
@@ -90,6 +95,36 @@ function M.shuffle(list)
   end
 end
 
+function M.logistic_scan()
+  local entities = game.surfaces[1].find_entities_filtered{name = {"logistic-chest-requester", "logistic-chest-buffer"}}
```

Comment:
Should this function loop through every surface?

Reply:
Probably.  I'm new, so I don't really understand multiple surfaces.